### DISCUSSION TITLE
LiTime Bluetooth BMS support

### DISCUSSION THREAD

**Discussion by aaron-baff-ad-net** [mr-manuel/venus-os_dbus-serialbattery#113] [New BMS requests]
So I recently got a LiTime Bluetooth battery, and trying to figure out how I might be able to reverse engineer to get the data off of it without their app, and came across https://github.com/chadj/litime-bluetooth-battery which does so via the Chrome Bluetooth extensions, and properly splits out cell voltages/etc. I emailed him asking if he had some documentation other than the JS code, but even if he doesn't it appears there's probably enough to go on to be able to pull out how to connect and extract the data.

It appears to be Bluetooth LE, which I haven't had a chance to read through how it's done on Linux in order to provide sample commands to connect/receive the payload and parse it out, but I should have some time in the near future.

Think it could be added as a supported BT BMS?

**Comment by mr-manuel** [comment]
Take a look at https://github.com/mr-manuel/venus-os_dbus-serialbattery/pull/59

**Reply by aaron-baff-ad-net** [reply]
@mr-manuel ah, thanks! I was looking only at the New BMS category, not all issues. Fantastic, thanks! I'll probably play around in a few weeks.

**Reply by mr-manuel** [reply]
The BMS was added.
